rskj-core/src/test/java/co/rsk/test/World.java

```diff
@@ -110,10 +111,22 @@ public World(
                 config.getActivationConfig());
     }
 
+    public ExecutionScopeFactory getExecutionScopeFactory() {
```

Comment:
Are you using this anywhere? Maybe you should be :stuck_out_tongue:

Reply:
Yup, it's being used by TraceModule